Spanner/tests/System/PgWriteTest.php

```diff
@@ -85,6 +87,9 @@ public function fieldValueProvider()
             [$this->randId(), 'floatfield', 3.1415],
             [$this->randId(), 'floatfield', INF],
             [$this->randId(), 'floatfield', -INF],
+            [$this->randId(), 'float4field', 3.1415],
+            [$this->randId(), 'float4field', INF],
+            [$this->randId(), 'float4field', -INF],
```

Comment:
How about we add one `null` also here?

Reply:
It's already being tested by `testWriteAndReadBackNullValue`